Management/src/ActuatorApi/Steeltoe.Samples.ActuatorApi.csproj

```diff
@@ -0,0 +1,68 @@
+    <PackageReference Include="OpenTelemetry.Instrumentation.Runtime" Version="$(OpenTelemetryVersion)" />
+    <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="$(EFCoreVersion)" />
+
+    <PackageReference Include="Steeltoe.Connectors.EntityFrameworkCore" Version="$(SteeltoeVersion)" />
+    <PackageReference Include="Steeltoe.Management.Prometheus" Version="$(SteeltoeVersion)" />
+    <PackageReference Include="Steeltoe.Management.Tracing" Version="$(SteeltoeVersion)" />
+    <PackageReference Include="Steeltoe.Management.Tasks" Version="$(SteeltoeVersion)" />
+
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.8.1" />
+    <PackageReference Include="System.Text.Json" Version="$(AspNetCoreVersion)" />
+  </ItemGroup>
+
+  <!--
+  In order to prevent build issues with processing a nonexistent .git folder (such as during staging in Cloud Foundry),
+  the following sections are only evaluated when the environment variable "CF_STACK" is NOT set.
+  -->
+  <ItemGroup Condition="'$(CF_STACK)' == ''">
+    <PackageReference Include="GitInfo" Version="3.5.0" PrivateAssets="all">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <Target Name="WriteGitPropertiesToFile" AfterTargets="CoreCompile" Condition="'$(CF_STACK)' == ''">
+    <!-- Generate git.properties file from the MSBuild properties provided by the GitInfo NuGet package. -->
+    <!-- https://github.com/devlooped/GitInfo/blob/main/src/GitInfo/build/GitInfo.targets -->
+    <WriteLinesToFile File="git.properties" Lines="git.remote.origin.url=$(GitRepositoryUrl)" Overwrite="true" />
+    <WriteLinesToFile File="git.properties" Lines="git.build.version=$(GitBaseVersion)" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.commit.id.abbrev=$(GitCommit)" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.commit.id=$(GitSha)" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.commit.time=$(GitCommitDate)" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.tags=$(GitTag)" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.branch=$(GitBranch)" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.build.time=$([System.DateTime]::Now.ToString('O'))" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.build.user.name=$([System.Environment]::GetEnvironmentVariable('USERNAME'))" Overwrite="false" />
+    <WriteLinesToFile File="git.properties" Lines="git.build.host=$([System.Environment]::GetEnvironmentVariable('COMPUTERNAME'))" Overwrite="false" />
+  </Target>
+
+  <ItemGroup Condition="'$(CF_STACK)' == ''">
+    <None Update="git.properties">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
This caused a 'huh?' moment for me, which is why I added an explaining comment. At first sight, it looks redundant because running with F5 doesn't need it.
It is *only* required for publish. Also, I used `Always` because `WriteGitPropertiesToFile` isn't conditional. So checking for changes first here is subobtimal.

> <!-- Ensure that the git.properties file ends up in the Publish directory. -->